tests/Spec/Nri/Ui/Panel.elm

```diff
@@ -0,0 +1,94 @@
+module Spec.Nri.Ui.Panel exposing (..)
+
+import Html
+import Html.Styled exposing (..)
+import Nri.Ui.Panel.V1 as Panel
+import ProgramTest exposing (..)
+import Test exposing (..)
+import Test.Html.Query as Query
+
+
+spec : Test
+spec =
+    describe "Nri.Ui.Panel.V1"
+        [ describe "renders content" rendersContent
+        ]
+
+
+rendersContent : List Test
+rendersContent =
+    [ test "renders markdown correctly" <|
+        \() ->
+            programMarkdown
+                |> ensureView
+                    (Query.contains
+                        [ Html.em [] [ Html.text "italic" ], Html.strong [] [ Html.text "bold" ] ]
+                    )
+                |> done
```

Comment:
I feel like this test isn't entirely self-contained -- you have to read more of the file and set up to understand what this test and the one after it are actually asserting.

I think inlining the setup, like so:
```
 \() ->
        ProgramTest.createSandbox
        { init = init
        , update = update
        , view =  
            \_ -> 
                 Panel.view
                    [ Panel.header "Header"
                    , Panel.markdown "_italic_ **bold**"
                    ]
                  |> toUnstyled
        }
                |> ProgramTest.start ()
                |> ensureView
                    (Query.contains
                        [ Html.em [] [ Html.text "italic" ], Html.strong [] [ Html.text "bold" ] ]
                    )
                |> done
```

Might help.

But really I think the biggest thing that might help is to _not_ use ProgramTest for these tests. Panel is not stateful, so I don't think we're getting a lot of extra value from ProgramTest, and it does add some boilerplate to set up.

```
test "renders markdown correctly" <|
        \() ->
            Panel.view
                [ Panel.header "Header"
                , Panel.markdown "_italic_ **bold**"
                ]
                |> toUnstyled
                |> Query.fromHtml
                |> Query.contains
                    [ Html.em [] [ Html.text "italic" ]
                    , Html.strong [] [ Html.text "bold" ]
                    ]
```